src/query.js

```diff
@@ -763,6 +768,8 @@ export default class Query {
     const json = this.toPlainObject();
     data = sift(json.filter, data);
 
+    Log.debug('After applying query filter to data', data);
```

Comment:
@thomasconner Have you considered that this data could be quite large and flood the log? Would it make sense to truncate or just print the count instead, for example?